crates/eips/src/eip1898.rs

```diff
@@ -556,6 +557,195 @@ impl FromStr for BlockId {
     }
 }
 
+/// Block number and hash.
+#[derive(Clone, Copy, Default, PartialEq, Eq, Hash)]
+pub struct BlockNumHash {
+    /// Block number
+    pub number: BlockNumber,
+    /// Block hash
+    pub hash: BlockHash,
+}
+
+/// Block number and hash of the forked block.
+pub type ForkBlock = BlockNumHash;
+
+impl fmt::Debug for BlockNumHash {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("").field(&self.number).field(&self.hash).finish()
+    }
+}
+
+impl fmt::Display for BlockNumHash {
```

Comment:
@mattsse what should the display impl be here?

Reply:
hmm, yeah actually display for this is opinionated, so we should remove.

I thought this could be useful in error types (reth) but better to just manually construct the hash+number error message

Reply:
should I remove fmt::Display altogether?